src/domain/tasks/AreaDropdownButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import { Area } from 'types/area'
+
+interface areasNamesAndPks {
+  name: string
+  pk: number
+}
+interface Props {
+  areas: areasNamesAndPks[]
+  addAreaFilter: (areaPk: number) => void
+}
+const AreaDropdownButton = ({ areas, addAreaFilter }: Props): JSX.Element => {
+  const [isOpen, setIsOpen] = useState(false)
+
+  return (
+    <div className="flex justify-center">
+      <div className="flex flex-row ">
+        <button onClick={(): void => setIsOpen(true)} className="block overflow-hidden">
+          <i className="icon-arrow-right text-disabled text-sm md:text-2xl"></i>
+        </button>
+        <div className="underline">Todas las areas</div>
```

Comment:
When there is an area selected, we need to show to the user the name of that area. "Todas las areas" could change to the area name if there is an area filter applied